docs/example.ts

```diff
@@ -378,7 +377,7 @@ async function doVerification(
     { senderDetails: verifierLightDid, receiverDetails: claimerLightDid }
   )
 
-  const ctypeHash = (verifierAcceptedClaimsMessageDec.body as IAcceptClaimsForCTypes)
+  const ctypeHash = (verifierAcceptedClaimsMessageDec.body as IAcceptCredential)
```

Comment:
M: Should we rename all occurrences of `claims` to `credential`?

Reply:
Not all, since the concept of a claim still exists, but in that case, it makes sense to rename.

Reply:
I was thinking about renaming e.g. `claimHash` to `credentialHash` in Attestation, but the chain is using that naming too, so we might want to look at naming for that at another time.

Reply:
renaming in example.ts file: 25f19097c467f9cbdbf07f95de33289b358c3a1d